PaulasWeatherApp/PaulasWeatherApp/ViewController/CurrentWeatherViewController.swift

```diff
@@ -8,11 +8,104 @@
 import UIKit
 
 class CurrentWeatherViewController: UIViewController {
+    
+    var tempLabel: UILabel = UILabel()
```

Comment:
Do you need to change these labels from the outside? If not (and this is the recommanded way to go) you should make all these properties private